tests/framework/utils_iperf.py

```diff
@@ -148,3 +148,21 @@ def consume_iperf3_output(stats_consumer, iperf_result):
 
     stats_consumer.consume_stat("Avg", CPU_UTILIZATION_VMM, vmm_util)
     stats_consumer.consume_stat("Avg", CPU_UTILIZATION_VCPUS_TOTAL, vcpu_util)
+
+    for idx, time_series in enumerate(iperf_result["g2h"]):
+        yield from [
+            (f"{THROUGHPUT}_g2h_{idx}", x["sum"]["bits_per_second"], "Megabits/Second")
+            for x in time_series["intervals"]
+        ]
```

Comment:
Just confirming, this is a single iteration, so we don't bother trying to preserve the timestamps and all the data will show up at the same time.

Reply:
Yep, that's how it is for now